cli/cmd/policy.go

```diff
@@ -183,6 +184,12 @@ func inputPolicy(cmd *cobra.Command) (string, error) {
 	if policyCmdState.URL != "" {
 		return inputPolicyFromURL(policyCmdState.URL)
 	}
+	// if running via stdin
+	stat, _ := os.Stdin.Stat()
```

Comment:
As a best practice, we always check errors, even if it means that we couldn't read the standard input. At least log a debug message.